tests/tests.py

```
@@ -412,6 +412,22 @@ def test_script_python():
     run(dpath("test_script_py"))
 
 
+@skip_on_windows
+@conda
```

Ruff (0.8.2)

Undefined name `conda`

(F821)